x-pack/plugins/security/server/session_management/session_index.ts

```diff
@@ -90,6 +90,7 @@ export function getSessionIndexSettings({
   indexName: string;
   aliasName: string;
 }): IndicesCreateRequest {
+  // @ts-expect-error ts upgrade to v4.7.4
```

Comment:
```suggestion

```

We can remove the `@ts-expect-error` and replace mappings with this:
 ```
    mappings: {
      dynamic: 'strict' as const,
      _meta: { [SESSION_INDEX_MAPPINGS_VERSION_META_FIELD_NAME]: SESSION_INDEX_MAPPINGS_VERSION },
      properties: {
        usernameHash: { type: 'keyword' as const },
        provider: { properties: { name: { type: 'keyword' as const }, type: { type: 'keyword'as const } } },
        idleTimeoutExpiration: { type: 'date' as const },
        createdAt: { type: 'date' as const },
        lifespanExpiration: { type: 'date' as const },
        accessAgreementAcknowledged: { type: 'boolean' as const },
        content: { type: 'binary' as const },
      },
    },
```

Reply:
Done 👍